### PR TITLE
new Contribution mechanism

### PR DESCRIPTION
**Pull request by ToyaSims**
Related issues: [please specify]

## Description:
this content doesn't have an issue, task was given during retro
What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,23 @@
...
-This syllabus is a Hugo based static site (for now). But on top of that, it is the configuration of our Tilde learning platform. You can see Tilde here: https://github.com/Umuzi-org/Tilde
+
+This syllabus is a Hugo based static site (for now). But on top of that, it is the configuration of our Tilde learning platform. You can see [Tilde](https://github.com/Umuzi-org/Tilde) for more.  
 
 I'm sure you've heard of infrastructure-as-code, pipeline-as-code and other x-as-code things? Well, this is a syllabus-as-code. Our learning platform eats this up and generates a bunch of trello-like cards that move across the personal Kanban boards of students using the platform.
 
 In order to make this work, we need to be strict on the shape of the syllabus files. We have very specific naming and metadata conventions that contributors need to follow. We'll talk more about that later. But let's start by getting you set up :)
 
-This video will walk you trough the whole process. Or you can just read the docs.
+This video will walk you trough the whole process. Or you can just read the [docs](https://www.youtube.com/watch?v=j5-uaSgIGI0&feature=youtu.be).
```
Comment (ry-oc) [inline README.md:14]: Docs is linking to a video

File: contribute.md [hunk 2]
```diff
@@ -1,18 +1,43 @@
...
-## Some guidelines
+
+## <a name="issues"></a> Choosing something to work on
+
+The first thing you should do is go to our [issues](https://github.com/Umuzi-org/ACN-syllabus/issues ) and look for something you might be interested in working on. The issues have what we call labels that help us distinguish between available and claimed issues, issues that are simple or hard and issues for hactoberfest, it is important to look at those labels before claiming and issue, labels an issue might have are:
+- status/help wanted
+- status/issue-claimed
+- good-first-issue
+- hactoberfest and 
+-experienced devs only
```
Comment (ry-oc) [inline contribute.md:30]: There should be a space for it to render correctly

File: contribute.md [hunk 3]
```diff
@@ -1,18 +1,43 @@
...
+
+if an issue has the `status/issue-claimed` label please skip it and if you a beginner it is advisable to start with the issues that have the `good-first-issue` label. You can assign yourself an issue by commenting your name on the comment section inside the issue, if you feel you need help with choosing an issue or you unsure of anything you can join our [discord channel](https://discord.com/channels/758708091583332382/762669571852468254) and you will get all the help you need.
```
Comment (ry-oc) [inline contribute.md:32]: Please change the discord link to an invite link not a channel link. You can use https://discord.gg/3jRwWPm

**Review by ry-oc — CHANGES_REQUESTED** [review]
See comments

File: README.md (hunk 1 above)
Comment (sheenarbw) [reply README.md:14]: bump

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Please finish this PR. There's feedback

**Review by asradebe — CHANGES_REQUESTED** [review]
Hi. Please attend to these requested changes by Friday so this can be merged. Please set up time on your calender to get this done

Thanks

**Comment by sheenarbw** [comment]
Oi. Shall we close this PR or what?

**Comment by ToyaSims** [comment]
No will work on it today

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**